app/controllers/job_postings_controller.rb

```diff
@@ -1,2 +1,37 @@
 class JobPostingsController < ApplicationController
+  def index
+    @job_posting = JobPosting.new
+  end
+
+  def new
+    @job_posting = JobPosting.new
+  end
+
+  def create
+    @job_posting = JobPosting.new(job_posting_params.merge(employer_id: current_employer.id))
+    if @job_posting.save
+      redirect_to root_path
+    else
+      render :index
```

Comment:
Note: For future reference, you may want to add an error message to the user and redirect them back to the new JobPosting page so they can correct their error.